.ebextensions/env-file-creation.config

```diff
@@ -0,0 +1,41 @@
+# Creates an .env file from AWS SSM Parameter Store
+
+commands:
+  01-create-env:
+    command: "/tmp/create-env.sh"
+
+files:
+  "/tmp/create-env.sh":
+      mode: "000755"
+      content : |
+        #!/bin/bash
+        # Reach into the undocumented container config
+        AWS_REGION='`{"Ref": "AWS::Region"}`'
+        ENV_NAME=$(jq .system.environment_name /opt/elasticbeanstalk/deploy/configuration/containerconfiguration | sed 's/"//g')
```

Comment:
@liangyuanruo and I had a chat abt this and we were thinking maybe there's a better way to do this. instead of coupling the ELB env name to the SSM parameters, can we just add an env var in ELB which indicates the prefix of the parameter stores? eg it can be `staging` or `prod`, and the staging parameter stores can be prefixed with `staging` and prod stores with `prod`